Makefile.am

```diff
@@ -115,15 +116,12 @@ SUBDIRS = .
 DIST_SUBDIRS = . perl/annotator perl/imap perl/sieve/managesieve
 dist_sysconf_DATA =
 lib_LTLIBRARIES = lib/libcyrus_min.la lib/libcyrus.la
-EXTRA_PROGRAMS =
 check_PROGRAMS =
 libexec_PROGRAMS =
 sbin_PROGRAMS =
 noinst_HEADERS =
 noinst_LTLIBRARIES =
 noinst_PROGRAMS =
-EXTRA_DIST = \
```

Comment:
Same comment as EXTRA_PROGRAMS

Reply:
There is no reason to include files in EXTRA_DIST under conditional paths: either the file is included by “make dist” or it is not included.  Hence a single EXTRA_DIST in Makefile.am for all relevant files is sufficient.